core/src/main/scala/org/apache/spark/status/api/v1/OneApplicationResource.scala

```diff
@@ -52,6 +53,25 @@ private[v1] class AbstractApplicationResource extends BaseAppResource {
   @Path("executors")
   def executorList(): Seq[ExecutorSummary] = withUI(_.store.executorList(true))
 
+  @GET
+  @Path("executorMetricsDistribution")
+  def executorSummary(
+      @QueryParam("activeOnly") @DefaultValue("true") activeOnly: Boolean,
+      @DefaultValue("0.05,0.25,0.5,0.75,0.95") @QueryParam("quantiles") quantileString: String)
```

Comment:
Why `0.05,0.25,0.5,0.75,0.95`? In stage page, Spark shows the quantiles of Min/Max

Reply:
To keep consistency with `/taskSummary` api in https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/status/api/v1/StagesResource.scala#L77, I used this default quantiles. 

And it seems stage page provided its own percentiles as below in https://github.com/apache/spark/blob/master/core/src/main/resources/org/apache/spark/ui/static/stagepage.js#L617

Reply:
FYI.  In the corresponding web UI, the quantiles of Min/Max are displayed in the table "Summary Metrics for Completed Tasks" for a given stage page.  In a parallel system, the duration of a stage is often determined by the slowest task/executor. To monitor/debug a skew issue, the maximal value (or 100% percentile value) is more useful than the 95% percentile value.  On the other hand, 95% percentile value has been used in the past.  One wise man once told me: Consistency means to repeat yesterday's mistake.